src/Microsoft.TemplateEngine.Orchestrator.RunnableProjects/LocalizableStrings.resx

```diff
@@ -160,7 +160,7 @@
     <value>Template: '{0}'</value>
   </data>
   <data name="Authoring_TemplateNotInstalled" xml:space="preserve">
-    <value>'{0}' was not loaded</value>
+    <value>Failed to load template from {0}:</value>
```

Comment:
It may be good to add comments to help translators understand what `{0}` is. I suppose it is a URL in this case?